torch_audiomentations/core/composition.py

```diff
@@ -0,0 +1,62 @@
+import random
+from typing import List
+
+import torch
+
+
+class Compose(torch.nn.Module):
+    """This class can apply a sequence of transforms to waveforms."""
+
+    def __init__(
+        self,
+        transforms: List[torch.nn.Module],
+        shuffle: bool = False,
+        p: float = 1.0,
+        p_mode="per_batch",
+    ):
+        """
+
+        :param transforms: List of waveform transform instances
+        :param shuffle: Should the order of transforms be shuffled?
+        :param p: The probability of applying the Compose to the given batch.
+        :param p_mode: Only "per_batch" is supported at the moment.
+        """
+        super().__init__()
+        self.transforms = transforms
```

Comment:
```suggestion
        self.transforms = torch.nn.ModuleList(transforms)
```
otherwise we'd loose differentiability, wouldn't we?
Maybe add a test in https://github.com/asteroid-team/torch-audiomentations/blob/master/tests/test_differentiable.py?

Reply:
Thanks, nice catch! Done.